libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/ScheduledEmailManagementDialog.tsx

```diff
@@ -36,30 +50,49 @@ export const ScheduledEmailManagementDialog: React.FC<IScheduledEmailManagementD
         setIsLoading(true);
     }, []);
 
+    const handleTabChange = useCallback((tab: ITab) => {
+        setSelectedTabId(tab.id);
+    }, []);
+
     useScheduledEmailManagement({
         loadScheduledMails: isLoading,
         onError: onLoadError,
         onSuccess: onLoadSuccess,
     });
 
+    const scheduledEmails = selectedTabId === ALL_TAB_ID ? scheduledEmailsAll : scheduledEmailsByUser;
+    const messageForEmptyId =
+        selectedTabId === ALL_TAB_ID
+            ? "dialogs.schedule.management.noSchedules.all"
+            : "dialogs.schedule.management.noSchedules.byUser";
+
     return (
         <>
             <Dialog
                 displayCloseButton={true}
                 onCancel={onClose}
                 className="gd-scheduled-email-management-dialog s-scheduled-email-management-dialog"
             >
-                <div>
-                    <Typography tagName="h3">
+                <div className="gd-scheduled-email-management-dialog-title">
+                    <Typography tagName="h3" className="gd-dialog-header">
                         <FormattedMessage id="dialogs.schedule.management.title" />
                     </Typography>
                 </div>
+                {!isFirstDataLoad && canManageScheduledMail && (
+                    <Tabs
+                        className="gd-scheduled-email-management-dialog-tabs"
+                        tabs={[{ id: BY_USER_TAB_ID }, { id: ALL_TAB_ID }]}
+                        selectedTabId={selectedTabId}
+                        onTabSelect={handleTabChange}
+                    />
+                )}
                 <div className="gd-scheduled-emails-content">
                     <ScheduledEmails
                         onDelete={handleScheduleDelete}
                         isLoading={isLoading}
                         scheduledEmails={scheduledEmails}
                         currentUserEmail={currentUser?.email}
+                        messageForEmpty={<FormattedMessage id={messageForEmptyId} values={{ br: <br /> }} />}
```

Comment:
Can we just pass the id and handle the translation in the child component? I would also name the prop accordingly like `noSchedulesMessageId`